package-lock.json

```diff
@@ -27979,22 +27979,6 @@
         "jest": "bin/jest.js"
       }
     },
-    "node_modules/jest-expo/node_modules/@babel/code-frame": {
```

Comment:
@BrtqKr Did this change got introduced unintended as these seem unrelated to our current scope here?

Reply:
I haven't modified anything in package.json and I'm using the appropriate node version. The change was introduced right after merging main to this branch and reinstalling the packages, so I'd assume it wasn't indended, but it's correct.

Reply:
This also doesn't seem like a blocker.